.gitignore

```diff
@@ -1,3 +1,12 @@
+## Ignore MATLAB artifacts
+*.asv
+
+## Ignore MATLAB data files
+*.mat
+
+## Ignore MATLAB compiler output
+*-compile/
+
```

Comment:
We won't compile MEX, nor use SIMULINK. I think those are not necessary. Possibly this one:
```
# OSX / *nix default autosave extension
*.m~
```
may be needed.